matrix_client/api.py

```diff
@@ -182,6 +182,11 @@ def logout(self):
         """
         return self._send("POST", "/logout")
 
+    def logout_all(self):
+        """Perform /logout/all.
+        """
```

Comment:
Put this on the first line so that it looks like `"""Perform /logout/all."""`. I know you looked at `logout`'s docstring, but this is the correct way :)